org.kde.okular.json

```diff
@@ -131,14 +131,14 @@
          ],
          "cleanup": [
             "/bin",
-            "/share/ghostscript/9.51/doc/",
-            "/share/ghostscript/9.51/examples"
+            "/share/ghostscript/9.52/doc/",
+            "/share/ghostscript/9.52/examples"
```

Comment:
Why not replace this with:
```
            "/share/ghostscript/*.*/doc",
            "/share/ghostscript/*.*/examples"
```

Reply:
I've no idea, i didn't write that. Maybe you can go try and see if there's something else worth keeping